homeassistant/components/trafikverket_camera/__init__.py

```diff
@@ -19,13 +19,15 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+TVCameraConfigEntry = ConfigEntry["TVDataUpdateCoordinator"]
```

Comment:
```suggestion
TVCameraConfigEntry = ConfigEntry[TVDataUpdateCoordinator]
```
The quotes aren't necessary here.